airbyte-cdk/java/airbyte-cdk/core/src/main/java/io/airbyte/cdk/db/jdbc/DefaultJdbcDatabase.java

```diff
@@ -73,6 +74,17 @@ public <T> Stream<T> unsafeResultSetQuery(final CheckedFunction<Connection, Resu
         });
   }
 
+  /**
+   * Implementations of DatabaseMetadata hold a reference of the Connection object. It is safe to use this to retrieve
```

Comment:
I think this comment only needs to exist on the interface, this implementation class should just inherit it?

Reply:
oh right. i wrote the impl first before interface 😅